### PR TITLE
catchup with embulk v0.10.32

### PR DESCRIPTION
**Pull request by legiangthanh**

### CODE REVIEW

**Review by vietnguyen-td — APPROVED** [review]
LGTM

File: src/main/java/org/embulk/input/zendesk/ZendeskInputPlugin.java [hunk 1]
```diff
@@ -240,7 +240,7 @@ public TaskReport run(final TaskSource taskSource, final Schema schema, final in
     @Override
     public ConfigDiff guess(final ConfigSource config)
     {
-        config.set("columns", new ObjectMapper().createArrayNode());
```
Comment (dmikurube) [inline src/main/java/org/embulk/input/zendesk/ZendeskInputPlugin.java:243]: Same as `embulk-input-jira` -- You may want to remove `import com.fasterxml.jackson.databind.ObjectMapper;` as well.

**Review by dmikurube — APPROVED** [review]
LGTM, and Reviewed for Data Exfiltrations.

Left one nitpicking comment.

File: src/main/java/org/embulk/input/zendesk/ZendeskInputPlugin.java (hunk 1 above)
Comment (legiangthanh) [reply src/main/java/org/embulk/input/zendesk/ZendeskInputPlugin.java:243]: ObjectMapper still use in some different place of this class.

**Pull request merged**